django/forms/utils.py

```diff
@@ -65,6 +65,11 @@ def as_ul(self):
             format_html_join('', '<li>{}{}</li>', self.items())
         )
 
+    def as_div(self):
```

Comment:
I'm not seeing the motivation for adding this method. 

* Why wouldn't `as_ul()` do? (Or just the existing `__str__()`  implementation.) 
* Given that the `admin/base.html` already wraps in a `ul`, would `as_text()` not do equally? (Are we really expecting multiple errors?)
* Why sure about `<b>` tags? 
* If we want this, tests?

Reply:
`as_div` is just a suggestion to provide a nicer view for users. 
`as_ul` makes the result view more separated. the inner `<li>` tags don't have error class therefore they seem with `icon-yes.svg`.
your're right, for this case `as_text` is more eligiable if we have just an error. I've changed with `as_text`.